src/components/Controls.vue

```diff
@@ -146,6 +167,20 @@ export default Vue.extend({
       a.click();
     },
 
+    search() {
+      const searchErrors: string[] = [];
+      if (this.network !== null) {
+        const nodeToSelect = this.network.nodes.find((node) => node._key === this.searchTerm);
```

Comment:
We should keep an eye on this line of code in case it causes slowdowns with larger networks.